docs/runtime/ir-caching.md

```diff
@@ -201,6 +202,19 @@ working state. This means that:
 At no point should this mechanism be exposed to the user in any visible way,
 other than the fact that they may be seeing the actual files on disk.
 
+### Imports
+
+Integrity Checking does not check the situation when the cached module imports a
```

Comment:
Git, BitCoin, etc. would try to solve this situation by including hashes of all imported modules into the hash of each module to create a _(block) chain_. Couldn't such solution be used in this case as well? Why?

Reply:
We could implement something like Merkle tree as well but it didn't seem necessary for the purpose of this fix.
Eventually, if we want to be more efficient in the kind of modules that we recompile we probably should combine hashes but I would say that this is a separate task.